### PR TITLE
Update readme to include match statement example

### PR DESCRIPTION
**Pull request by unique1o1**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -61,9 +61,25 @@ To something like this:
...
         raise RuntimeError('Could not fetch user: %s' % user_result.value)
 
-As this is Python and not Rust, you will lose some of the advantages that it
-brings, like elegant combinations with the ``match`` statement. On the other
-side, you don't have to return semantically unclear tuples anymore.
+And if you're using python version  ``3.10``, you can use the elegant ``match`` statement as well:
+
+.. sourcecode:: python
+
+    from rusty_results import Result, Ok, Err
```
Comment (francium) [inline README.rst:68]: I think you mixed up this package with another :)
`rusty_results` -> `result`

**Comment by francium** [comment]
Just one small change needed. Thanks for the PR!

File: README.rst [hunk 2]
```diff
@@ -61,9 +61,25 @@ To something like this:
         # type(user_result.value) == str
         raise RuntimeError('Could not fetch user: %s' % user_result.value)
 
-As this is Python and not Rust, you will lose some of the advantages that it
-brings, like elegant combinations with the ``match`` statement. On the other
-side, you don't have to return semantically unclear tuples anymore.
+And if you're using python version  ``3.10``, you can use the elegant ``match`` statement as well:
```
Comment (francium) [inline README.rst:64]: Should probably reword this too,
`And if you're using Python 3.10 or newer`

**Comment by unique1o1** [comment]
Is the pattern matching support from #48 not been uploaded to PyPI?

**Review by francium — APPROVED** [review]

**Comment by francium** [comment]
> Is the pattern matching support from #48 not been uploaded to PyPI?

No, we're overdue for a release. I don't have permissions to make a release. I'll talk with the repo owner to get a release out.

**Comment by francium** [comment]
@unique1o1 thanks for the PR
